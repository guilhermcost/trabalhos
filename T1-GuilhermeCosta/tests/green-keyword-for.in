int main(void) {
int i;

for (i=1; i<10; i=i+1)
{
	println(i);
}

}
