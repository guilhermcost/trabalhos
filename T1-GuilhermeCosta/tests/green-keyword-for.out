(1,KEY,"int")
(1,ID,"main")
(1,SYM,"(")
(1,KEY,"void")
(1,SYM,")")
(1,SYM,"{")
(2,KEY,"int")
(2,ID,"i")
(2,SYM,";")
(4,KEY,"for")
(4,SYM,"(")
(4,ID,"i")
(4,SYM,"=")
(4,NUM,"1")
(4,SYM,";")
(4,ID,"i")
(4,SYM,"<")
(4,NUM,"10")
(4,SYM,";")
(4,ID,"i")
(4,SYM,"=")
(4,ID,"i")
(4,SYM,"+")
(4,NUM,"1")
(4,SYM,")")
(5,SYM,"{")
(6,ID,"println")
(6,SYM,"(")
(6,ID,"i")
(6,SYM,")")
(6,SYM,";")
(7,SYM,"}")
(9,SYM,"}")
